src/test/java/doit/jpastudy2/Library/repository/BookRepositoryTest.java

```diff
@@ -0,0 +1,32 @@
+package doit.jpastudy2.Library.repository;
+
+import doit.jpastudy2.Library.Book;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional
+@SpringBootTest
+class BookRepositoryTest {
+
+    @Autowired
+    BookRepository bookRepository;
+
+    @Test
+    void test(){
+        Book book = Book.builder()
+                .title("모순")
+                .author("양귀자")
+                .isbn("9788998441012")
+                .publisher("쓰다")
+                .build();
+
+        bookRepository.save(book);
+
+        Book book1 = bookRepository.findByTitleAndAuthor("모순","양귀자");
+
+        System.out.println("Member Id = " + book1.getBookId());
```

Comment:
```suggestion
    Assertions.assertThat(book1).isNotNull();
    Assertions.assertThat(book1.getTitle()).isEqualTo("모순");
    Assertions.assertThat(book1.getAuthor()).isEqualTo("양귀자");
    Assertions.assertThat(book1.getIsbn()).isEqualTo("9788998441012");
    Assertions.assertThat(book1.getPublisher()).isEqualTo("쓰다");
```
종종 직접 값에 대해 print하는 것도 눈으로 확인하고 더욱 확신이 가고 좋습니다.
다만 위처럼 Assertions를 이용하여 값에 대한 검증을 프로그램에게 넘긴다면, 여러 개의 테스트 케이스를 확인하는데 훨씬 편리하다는 점 참고하시면 될 것 같습니다!! 고생하셨습니다.